frontend/src/components/AccountReceiveTable.vue

```diff
@@ -473,7 +487,7 @@ function useReceivedFundsTable(announcements: UserAnnouncement[], spendingKeyPai
 
     // Format addresses to use ENS, CNS, or formatted address
     const fromAddresses = announcements.map((announcement) => announcement.from);
-    const formattedAddresses = await lookupOrFormatAddresses(fromAddresses, MAINNET_PROVIDER as Web3Provider);
+    const formattedAddresses = await lookupOrReturnAddresses(fromAddresses, MAINNET_PROVIDER as Web3Provider);
```

Comment:
Why was this change made? It seems we added a method which is similar to `lookupOrFormatAddress` but doesn't do the formatting, e.g. it returns the full address and not a truncated version?

If we keep this change, it means we no longer use `lookupOrFormatAddress[es]` and should delete those

Reply:
I changed it because the sender's address `announcement.from` was actually being overwritten by the truncated version here https://github.com/ScopeLift/umbra-protocol/blob/34f45f3ce389d5aecc603814adfadcab5c18758a/frontend/src/components/AccountReceiveTable.vue#L476-L479 The copy address button would end up copying the truncated version of the address and the ehterscan link returned 404 because of it. So I ended up formatting inside the template before displaying. https://github.com/ScopeLift/umbra-protocol/blob/32dbe9b5356ef0b55603100c9cbdffd44233a174/frontend/src/components/AccountReceiveTable.vue#L238

To solve the ENS truncation problem tho, should I maybe create a new announcement value that stores a formatted and an unformatted version of addresses so I can link to unformatted ones but show the formatted ones?

Reply:
> I changed it because the sender's address announcement.from was actually being overwritten by the truncated version here
> 
> The copy address button copied the truncated version of the address and the ehterscan link returned 404 because of it. So I ended up formatting inside the template before displaying.

This all makes sense.

> To solve the ENS truncation problem tho, should I maybe create a new announcement value that stores a formatted and an unformatted version of addresses so I can link to unformatted ones but show the formatted ones?

The problem here is that you do `formatAddress(col.value)`, but sometimes `col.value` is an ENS name, and `formatAddress` truncates and adds `...` regardless of whether its a hex string or ENS name. So it seems you can just make that method more robust and do something like this:
- if the ethers.js `isHexString` method returns true, it's an address so use the current logic
- otherwise, it's not an address so do nothing and return the full ENS name.

Then I think we can delete the lookupOrFormat methods that we no longer use and then we should be all set

Reply:
Addressed it here https://github.com/ScopeLift/umbra-protocol/pull/381/commits/8cc736ac2fcdf389065d9a482f1ed854f2a04bb4